x/bank/app_test.go

```diff
@@ -212,7 +214,8 @@ func TestMsgMultiSendMultipleOut(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		mock.SignCheckDeliver(t, mapp.Cdc, mapp.BaseApp, tc.msgs, tc.accNums, tc.accSeqs, tc.expSimPass, tc.expPass, tc.privKeys...)
+		header := abci.Header{Height: mapp.LastBlockHeight() + 1}
+		mock.SignCheckDeliver(t, mapp.Cdc, mapp.BaseApp, header, tc.msgs, tc.accNums, tc.accSeqs, tc.expSimPass, tc.expPass, tc.privKeys...)
```

Comment:
I've noticed this line addition `header := abci.Header{Height: mapp.LastBlockHeight() + 1}` in most of the tests - this is performed in the CLI already correct?

Reply:
yes @rigelrozanski, the header is properly set by Tendermint.